pfcpiface/pfcpsim.go

```diff
@@ -253,7 +253,7 @@ func modifyPFCP(conn *net.UDPConn, raddr *net.UDPAddr, seid uint64) {
 			seid,
 			seq,
 			0,
-			//ie.NewFSEID(0x0000000000000001, net.ParseIP("127.0.0.1"), nil),
+			//ie.NewFSEID(0x0000000000000001, net.GetUnicastAddressFromInterface("127.0.0.1"), nil),
```

Comment:
Should we remove this line since it's commented out?

Reply:
sure, we're moving away from this sim mode anyway.